### ISSUE TITLE
Mega link not working 

### ISSUE THREAD

**Issue by albaraathunder** [Vexed-Nz/DiscordTokenGen#1]
The title says everything really. You can use git to upload the files even if you cant upload using the website because it is too large.
[Git download](https://git-scm.com/downloads)

**Comment by Vexed-Nz** [comment]
Yeah sorry I about that I discontinue the project since there were some errors in the Python Script I used it before and made tokens in 10min. If you want to talk more Join Our discord = https://discord.gg/Pv2VzS4XbS
